### ISSUE TITLE
Problems with CS:GO

### ISSUE THREAD

**Issue by skyne98** [labwc/labwc#120]
First of all, awesome project! It's refreshing to finally have a reasonably stable and simple *stacking* compositor for Wayland.

So, I have noticed a couple of issues:
1. When you launch CS:GO from Steam, in both Fullscreen and Windowed Fullscreen mode the game is not able to get rid of the decorations by itself and take the whole screen.
2. It seems like the game is completely unable to lock the mouse cursor, turning the camera input into a complete mess.
3. (A little sidetrack) It also seems like you are unable to move around the Steam's window using its handle for some reason.

Maybe I just configured something wrong, but this is how it is for me currently (with the default example 'rc.xml').

**Comment by Joshua-Ashton** [comment]
All these bugs are already fixed. Are you running some supee outdated version?

**Comment by skyne98** [comment]
Ah, my bad, I was running from [this AUR pacakge](https://aur.archlinux.org/packages/labwc/) and didn't check the date of last update. I will now update to the latest version and check all the issues I have described!

Thanks for the heads up!

**Comment by skyne98** [comment]
Yep, I can confirm that the issues I described above are solved. However, I found a couple of other things: 
1. CS:GO refuses to open in full screen on the second screen. Actually, when you force it to ([using this method with `sdl_displayindex`](https://github.com/ValveSoftware/csgo-osx-linux/issues/60), it's an old engine, but this method works on GNOME) then it still displays on the first display, but using the resolution of the second one (takes only a part of the screen, in case second screen has lower resolution).
2. In general, I have noticed that games running through Proton (Experimental) either completely fail to display in a proper window or take a whole screen. In case of games like DOOM: Eternal, which runs on Vulkan, choosing the display directly in the settings didn't properly change the display.
3. (Minor Issue) When Steam launches for the first time, I have noticed that hovering over menus (top ones) displays the popup on the second screen. Moving the screen a couple of pixels snaps them back, and afterwards it is displayed properly. I wonder if this is even the compositor's issue and not Steam's.
